### DISCUSSION TITLE
Referencing code and module in my blog post

### DISCUSSION THREAD

**Discussion by Gwiazda01** [d365collaborative/d365fo.tools#651] [Q&A] [answered]
Hi,
I want to ask if it is ok for You to reference this powershell module and github site in my own blog post? I'm not 100% sure that under MIT license I could do it without asking authors and could not reach You any other way. 
I want to reference this module and site for something like "usefull links". I'm waiting for response :)
Regards, 
Kamil

**Comment by Splaxi** [comment, marked as answer]
Sure thing 🤘

The MIT license allows you to do whatever you want, but at your own risk 😉

Ping me on Twitter (@splaxi) and I'll promote your blog post/page.

**Reply by Gwiazda01** [reply]
Thanks, of course I'll ping you. I appreciate your proposal regarding promotion, but please do this only if you will rate my blog post high enough 🙂
